assistant/src/main/java/com/ibm/watson/assistant/v1/model/DialogNodeOutputGenericDialogNodeOutputResponseTypeAudio.java

```diff
@@ -157,6 +157,8 @@ public Builder altText(String altText) {
     }
   }
 
+  protected DialogNodeOutputGenericDialogNodeOutputResponseTypeAudio() {}
```

Comment:
I see that this type of empty protected class is happening everywhere, I think that's part of the new version for the generator